helm/circles-infra-suite/templates/configmap.yaml

```diff
@@ -60,3 +60,7 @@ data:
   DJANGO_ALLOWED_HOSTS: .{{.Values.host}}
   DJANGO_SETTINGS_MODULE: config.settings.production
   RELAYER_REDIS_URL: redis://{{.Values.redisService.name}}:{{.Values.redisService.port}}/0
+
+  # Payment API Configuration
+  CHAIN_ID: '{{.Values.apiPaymentService.chaindID}}'
```

Comment:
The `apiPaymentService.chaindID` value has already single quotes, even i do not if they are neessary. Here, single quotes are used again.

Reply:
removed